services/worker/src/worker/job_runners/split/descriptive_statistics.py

```diff
@@ -527,19 +526,24 @@ def compute_descriptive_statistics_response(
             "no features found in 'dataset_info'. "
         )
 
-    split_parquets_size = sum(parquet_file["size"] for parquet_file in split_parquet_files)
-    if split_parquets_size > max_parquet_size_bytes:
-        raise SplitWithTooBigParquetError(
-            f"Statistics computation is limited to split parquets under {max_parquet_size_bytes} bytes. "
-            f"Current size of sum of split parquets is {split_parquets_size} bytes."
-        )
+    num_parquet_files_to_index = 0
+    total_bytes = 0
+    split_directory = split_parquet_files[0]["url"].rsplit("/", 2)[1]
+    partial_parquet_export = parquet_export_is_partial(split_parquet_files[0]["url"])
+    for parquet_file_id, parquet_file in enumerate(split_parquet_files):
+        num_parquet_files_to_index += 1
+        parquet_file_size_bytes = parquet_file["size"]
+        total_bytes += parquet_file_size_bytes
+        if total_bytes > max_dataset_size_bytes:
```

Comment:
it means that we will compute the stats for more than the limit, right? (off by one issue?)

Reply:
oops i just saw this comment. yes! this is copied from duckdb-index. that's why i was struggling so much with the description :D but now I just put the word `"approximately"` everywhere, i think it's enough: d9c11fb, e1c45af

Reply:
ok